src-docs/src/views/date_picker/date_picker_example.js

```diff
@@ -254,16 +255,25 @@ export const DatePickerExample = {
         },
       ],
       text: (
-        <p>
-          Locale formatting is achieved by using the <EuiCode>locale</EuiCode>,
-          <EuiCode>timeFormat</EuiCode>, and <EuiCode>dateFormat</EuiCode>{' '}
-          props. The latter will take any <EuiCode>moment()</EuiCode> notation.
-          Check{' '}
-          <a href="https://en.wikipedia.org/wiki/Date_format_by_country">
-            Date format by country
-          </a>{' '}
-          for formatting examples.
-        </p>
+        <>
+          <p>
+            Locale formatting is achieved by using the <EuiCode>locale</EuiCode>
+            ,<EuiCode>timeFormat</EuiCode>, and <EuiCode>dateFormat</EuiCode>{' '}
+            props. The latter will take any <EuiCode>moment()</EuiCode>{' '}
+            notation. Check{' '}
+            <a href="https://en.wikipedia.org/wiki/Date_format_by_country">
+              Date format by country
+            </a>{' '}
+            for formatting examples.
+          </p>
+          <EuiCallOut color="warning">
+            Moment will try to load the locale on demand when it is used.
+            Bundlers that do not support dynamic imports will need to explicitly
```

Comment:
```suggestion
            Bundlers that do not support dynamic require statements will need to explicitly
```

I think this nit could be meaningful - moment uses a `require()` statement and that's likely starting to matter as the ecosystem shifts more to ESM.

Reply:
https://github.com/elastic/eui/pull/6234/commits/84c1af0bb168fa647a56758bd2bc64be4817da83